### PR TITLE
Adding Flask cheatsheet to compliment Python Web Framework resources

### PR DESCRIPTION
**Pull request by Targgus**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Targgus** [comment]
Corrected spacing between subheading and resource link

**Review by LuigiImVector — APPROVED** [review]

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -307,6 +308,10 @@
 * [Django Web App Beginners Cheat sheet](https://ordinarycoders.com/blog/article/django-beginners-guide) - Jaysha (HTML)
 * [(Python + Django) Cheatsheet](https://dev.to/ericchapman/my-beloved-django-cheat-sheet-2056) - Eric The Coder (HTML)
```
[the remark this answers is not in the document]
Comment (LuigiImVector) [reply more/free-programming-cheatsheets.md:310]: up

**Comment by Targgus** [comment]
Found the space I was missing and this is ready for merge. Thanks!

**Comment by eshellman** [comment]
Go team

**Pull request merged**